service/operator/resource/configmap/service.go

```diff
@@ -0,0 +1,289 @@
+package configmap
```

Comment:
Before there have been both resources together in the implemented `Operator` interface. Now the config map resource is separated into its own `Resource` implementation. Besides moving code into its own package and adapting to the import changes of the k8s client due to vendor updates nothing changed. Tests have been adapted as well.